openfecwebapp/api_caller.py

```diff
@@ -34,7 +37,11 @@ def _call_api(*path_parts, **filters):
 
     results = session.get(url, params=filters)
 
-    return results.json() if results.ok else {}
+    if results.ok:
+        return results.json()
+    else:
+        logger.error('API ERROR with status {0} for {1} with filters: {2}'.format(results.status_code, url, filters))
```

Comment:
This will give enough of a clue to know what to check for on the API side, but would it make sense to also include the exception message thrown by the API too, or do you think that's not necessary here?

Either way, this definitely provides much better visibility into a request that failed when going through the `api_caller`!